src/app/(proper_react)/(redesign)/(public)/LandingView.stories.tsx

```diff
@@ -15,6 +15,8 @@ const meta: Meta<typeof View> = {
   component: (props: ViewProps) => {
     const experimentData =
       props.experimentData ?? defaultExperimentData["Features"];
+    const { enabledFeatureFlags: enabledFeatureFlagsProp } = props;
+    const enabledFeatureFlags = enabledFeatureFlagsProp ?? [];
```

Comment:
Heh, am I misreading this, or is this a somewhat convoluted way of doing

```suggestion
    const enabledFeatureFlags = props.enabledFeatureFlags ?? [];
```

? :sweat_smile:

Reply:
I just enjoy destructering too much and went overboard here. Updated in [0b448ba](https://github.com/mozilla/blurts-server/pull/5717/commits/0b448baa87b18d4a760fcf37d9731865b8313fc1).